libraries/AP_Mount/AP_Mount_Servo.cpp

```diff
@@ -191,11 +179,11 @@ void AP_Mount_Servo::stabilize()
 // closest_limit - returns closest angle to 'angle' taking into account limits.  all angles are in degrees * 10
 int16_t AP_Mount_Servo::closest_limit(int16_t angle, int16_t angle_min, int16_t angle_max)
 {
-    // Make sure the angle lies in the interval [-180 .. 180[ degrees
+    // Make sure the angle lies in the interval [-180 .. 180] degrees
     while (angle < -1800) angle += 3600;
     while (angle >= 1800) angle -= 3600;
 
-    // Make sure the angle limits lie in the interval [-180 .. 180[ degrees
+    // Make sure the angle limits lie in the interval [-180 .. 180] degrees
     while (angle_min < -1800) angle_min += 3600;
```

Comment:
The old comment has the correct formal mathematical notion, please revert this

Reply:
ok, reverted